### DISCUSSION TITLE
Confused On Which To Use For Sending/Fetching Data

### DISCUSSION THREAD

**Discussion by kecoliva** [nuxt/nuxt#21902] [Questions]
I don't know whether should I use `$fetch` or `useFetch` for sending/fetching data from the client. When should I use `$fetch` and when should I use `useFetch` and when should I choose one over the other?

**Comment by AloisSeckar** [comment]
Recently I have read a great summarization about this topic https://mokkapps.de/blog/a-comprehensive-guide-to-data-fetching-in-nuxt-3
